src/tests/stdlib/is_root.ab

```diff
@@ -0,0 +1,6 @@
+import { is_root } from "std"
+main {
+    if not is_root() {
+        echo "no"
+    }
```

Comment:
This would fail if ran as root. Can we run a command to check if user is root and then match the results instead?

Reply:
I tested the ab script as root and works, if it is root doesn't print anything.